google/cloud/bigquery/dtypes.py

```diff
@@ -0,0 +1,471 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import datetime
+import operator
+from typing import Any, Optional, Sequence
+
+import numpy
+import packaging.version
+import pandas
+from pandas._libs import NaT
+from pandas._libs.lib import is_integer
+from pandas._typing import Scalar
+import pandas.compat.numpy.function
+import pandas.core.algorithms
+import pandas.core.arrays
+import pandas.core.dtypes.base
+from pandas.core.dtypes.common import is_dtype_equal, is_list_like, pandas_dtype
+import pandas.core.dtypes.dtypes
+import pandas.core.dtypes.generic
+import pandas.core.nanops
+import pyarrow
+
+pandas_release = packaging.version.parse(pandas.__version__).release
```

Comment:
The `.release` attribute was added in packaging 17.0. We'll have to bump the minimum in `setup.py` (and add a constraint, since it seems we missed this)

https://github.com/pypa/packaging/blob/e2bc8f6ef2d66f6eaaf522fcf70a0f203b734064/CHANGELOG.rst#170---2017-02-28

I'm addressing this in https://github.com/googleapis/python-db-dtypes-pandas/pull/1